tests/block.go

```diff
@@ -225,9 +225,9 @@ func (tp *TestSuite) TestBlockPin(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	pins, err := api.Pin().Ls(ctx)
+	pins, err := accPins(api.Pin().Ls(ctx))
 	if err != nil {
-		return
+		t.Skip(err)
```

Comment:
This is problematic. Before this PR, the test was just terminating and didn't actually test anything.

With this workaround, running the test suite in go-ipfs produce `block.go:230: failed to decode Protocol Buffers: incorrectly formatted merkledag node: unmarshal failed. proto: PBNode: wiretype end group for non-group`.

I don't know what the root cause is.

Reply:
We're putting a bad block...